src/java/org/apache/cassandra/db/DeletionTime.java

```diff
@@ -121,7 +120,7 @@ public void digest(Digest digest)
      */
     public boolean validate()
     {
-        return true;
+        return markedForDeleteAt >= 0 || isLive();
```

Comment:
I would amend the comment of the method. Mentioning a negative ldt and that not being checked is confusing imo.

Reply:
Done